skopt/solvers/paramils.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+
+from sklearn.externals.joblib import Parallel, delayed
+from sklearn.utils import check_random_state
+
+def _transform_back(x0, lower_bounds, upper_bounds):
+    return x0 * (upper_bounds - lower_bounds) + lower_bounds
+
+def _local_search(func, x0, bounds, std, n_neighbors,
+                  random_state=None, maxiter=10):
+    rng = check_random_state(random_state)
+    upper_bounds = bounds[:, 1]
+    lower_bounds = bounds[:, 0]
+
+    transformed_x_min = np.asarray(x0)
+    actual_x_min = _transform_back(
+        transformed_x_min, lower_bounds, upper_bounds)
+
+    fun_min = func(actual_x_min.reshape(1, -1))[0]
+    n_dimensions = len(x0)
+
+    for i in range(maxiter):
+        # TODO: Cache and modify in-place.
+        transformed_points = np.tile(
+            transformed_x_min, (n_neighbors * n_dimensions, 1))
+
+        for dim_ind in range(n_dimensions):
+
+            # Sample n_neighbors number of points from a univariate
+            # gaussian with the previous minimum as the mean
+            # and provided standard deviation.
+            random_points = np.clip(
+                transformed_x_min[dim_ind] + std * rng.randn(n_neighbors),
+                0, 1)
+
+            start_ptr = dim_ind * n_neighbors
+            end_ptr = start_ptr + n_neighbors
+            transformed_points[start_ptr: end_ptr, dim_ind] = random_points
+
+        actual_points = _transform_back(
```

Comment:
@glouppe I think I lost your comment here. Did you mean handling the transform back into the original space inside the acquisition function?

In that case, we will not be able to use this standalone solver right?
Or did you mean something else?